core/src/components/checkbox/test/item/checkbox.e2e.ts

```diff
@@ -69,6 +69,22 @@ configs({ directions: ['ltr'], modes: ['md'] }).forEach(({ title, screenshot, co
       const list = page.locator('ion-list');
       await expect(list).toHaveScreenshot(screenshot(`checkbox-long-label-in-item`));
     });
+    test('should render margins correctly when using long label in item with start alignment', async ({ page }) => {
```

Comment:
Should we test this in ios mode as well based on your comment in the PR about md styling still being slightly off?

Reply:
I thought about that. I actually added `ios` mode but then removed it because the styles applied are the same for both modes since they both use `checkbox.scss`. I can add `ios` too though if we just want the extra visual checks.

Reply:
I'll defer to your judgement, but just thought I'd bring it up

Reply:
It doesn't add many screenshots, so I added it in https://github.com/ionic-team/ionic-framework/pull/29850/commits/0ae6ae2c139862565f4a77d105a7844dd83e1936 and https://github.com/ionic-team/ionic-framework/pull/29850/commits/7862ef41874144d61930ebfe1c20dae76ea14401